tests/reg_tests/test_DVConstraints.py

```diff
@@ -69,7 +69,7 @@ def evalFunctionsSensFD(self, DVGeo, DVCon, fdstep=1e-2):
                     funcsSens[outkey][inkey][:, array_ind] = deriv_temp
                 xDV[inkey][array_ind] = baseVar[array_ind]
         DVGeo.setDesignVars(xDV)
-        DVCon.evalFunctions(dict())
+        DVCon.evalFunctions({})
```

Comment:
I've no idea why this line is needed..

Reply:
You mean `{}` instead of `dict()`? Or the fact that we are calling `DVCon`?

Reply:
Oh now I see, it's a third call to `DVCon` at the end, just before exiting the FD sensitivity method.. Maybe it's just to reset the internal values, same as done by `DVGeo.setDesignVars()` above? I need to check the function

Reply:
@nwu63 I suspect that it is needed because every time we evaluate a constraint we [update the coordinates of the pyGeo object](https://github.com/mdolab/pygeo/blob/9eff5c04b999c8e8f7e99c80b6850868a87415c3/pygeo/constraints/thicknessConstraint.py#L51). I think that if we directly use `DVGeo.update()` the tests should work, up to you if you want to play with that.
I might be wrong bc I don't know what error you are getting.

Reply:
Yeah I've no idea, but without line 72 one of the tests fail, and if I also take out line 71 then a few more tests fail. I can see cases where 71 is needed (even though it's probably not the best practice), but not sure why 72 is needed at all.

Reply:
I can't tell what that line is doing either, the error I get from taking it out is in `Test 8: Circularity constraint` and looks like it's not meeting the tolerance. Neil is this what you're seeing?
```
AssertionError: 
Not equal to tolerance rtol=1e-07, atol=1e-07
Failed value for: circularity_base
Mismatched elements: 8 / 9 (88.9%)
Max absolute difference: 2.15268112e-05
Max relative difference: 2.15268112e-05
 x: array([0.999989, 0.999978, 0.999987, 1.      , 0.999995, 0.99998 ,
       0.999981, 0.999994, 1.      ])
 y: array([1., 1., 1., 1., 1., 1., 1., 1., 1.])
```

Reply:
Yes exactly.